TypeCobol/Compiler/CompilationDocument.cs

```diff
@@ -643,9 +650,6 @@ public void RefreshProcessedTokensDocumentSnapshot()
                 // Start perf measurement
                 var perfStatsForParserInvocation = PerfStatsForPreprocessor.OnStartRefreshParsingStep();
```

Comment:
This must be called at the beginning of the method.
Method `GetReducedAndOrderedChangesInNewerVersion` use iterator on the `ImmutableList` of Lines and it may be slow.

Reply:
ok done